src/OpenLoco/src/GameCommands/GameCommands.h

```diff
@@ -719,6 +778,47 @@ namespace OpenLoco::GameCommands
         doCommand(GameCommand::changeCompanyColourScheme, regs);
     }
 
+    struct ChangeCompanyColourSchemeArgs
+    {
+        ChangeCompanyColourSchemeArgs() = default;
```

Comment:
dont forget to add  `static constexpr auto command = GameCommand::changeCompanyColourScheme` goes for a good few of these structs.

Reply:
Sorry, my bad

Reply:
Couldn´t we extend the structs from an abstract one to require the command member be required? Would help in these cases